rest/rest-server-jersey/src/test/java/org/apache/brooklyn/rest/BrooklynRestApiLauncherTest.java

```diff
@@ -36,17 +36,17 @@
 
 public class BrooklynRestApiLauncherTest extends BrooklynRestApiLauncherTestFixture {
 
-    @Test
+    @Test(enabled = false) // to be deleted
```

Comment:
Yes, but doesn't really matter because already deleted in https://github.com/apache/brooklyn-server/pull/95.